server/app.js

```diff
@@ -2,10 +2,11 @@
 // Set working directory to project root
 import { dirname }       from 'path';
 import { fileURLToPath } from 'url';
-import packageJSON       from './../package.json' with { type: 'json' };
 
 const __dirname = dirname(fileURLToPath(import.meta.url));
 process.chdir(`${__dirname}/..`);
+
+import packageJSON from '../package.json' with { type: 'json' };
```

Comment:
Just checking why this import was moved

Reply:
That was moved during some troubleshooting; I've reverted it now.